packages/ag-charts-types/src/chart/toolbarOptions.ts

```diff
@@ -74,7 +74,6 @@ export interface AgToolbarAnnotationsGroup extends AgToolbarGroup {
 }
 
 export interface AgToolbarAnnotationsButton extends AgToolbarButton {
-    section?: 'create' | 'tools';
```

Comment:
This is the expanding of the type I mentioned yesterday. It is now `section?: string` instead.